charts/policy-controller/Chart.yaml

```diff
@@ -8,8 +8,8 @@ sources:
 type: application
 
 name: policy-controller
-version: 0.2.1
-appVersion: 0.2.1
+version: 0.2.2
+appVersion: 0.2.2
```

Comment:
The policy-controller version no longer matches the latest released version https://github.com/sigstore/policy-controller/releases/tag/v0.2.1

I'd suggest leaving these versions as they are. These will be changed once there's a release.

Reply:
It is only policy controller that doesn’t? 

The GitHub PR template says to update it so I want to update to reflect properly.

Reply:
reverted version number and minor adjustment to github template.

Reply:
The workflow doesn't allow not bumping the version.
```
policy-controller => (version: "0.2.1", path: "charts/policy-controller") > Chart version not ok. Needs a version bump!
```

Reply:
@JAORMX Any idea how to resolve this?   The pipeline fails if the version is not bumped

Reply:
@slim609 I don't know. I suggest you ask on the slack channel(s)